server/routes/messageRoutes.js

```diff
@@ -8,7 +8,52 @@
 
 const express = require('express');
 const router = express.Router();
+const { validationResult } = require('express-validator/check');
+const { isLoggedIn, userIsMember } = require('./routeUtils');
+const message = require('../controllers/messagesController');
+const messageValidator = require('../controllers/validators/messageControllerValidator');
 
+router.post('/create/',
+    isLoggedIn,
+    userIsMember,
+    messageValidator.validate('checkMessage'),
+    function (req, res){
+        const errors = validationResult(req);
 
+        if (!errors.isEmpty()) {
+            res.status(202).json({ errors: errors.array()});
+        } else {
+            message.createMessage(req,req.user.uid)
+                .then( (message) => res.status(200).json({
+                    senderId: message.senderId,
+                    receiverId:message.receiverId,
+                    content: message.content,
+                }))
```

Comment:
Is there a specific reason that you are sending back the information to the user? A simple boolean flag would also work and would be simpler to work with on the front-end
```
{
    sent: true
}
```

Reply:
Addressed.